ebos/damariswriter.cc

```diff
@@ -81,4 +85,62 @@ int endIteration(int rank)
     return dam_err;
 }
 
+int setupWritingPars(Parallel::Communication comm,
+                     const int n_elements_local_grid,
+                     std::vector<unsigned long long>& elements_rank_offsets)
+{
+    // one for each rank -- to be gathered from each client rank
+    std::vector<unsigned long long> elements_rank_sizes(comm.size());
```

Comment:
I guess we get a declaration of `vector<>` from somewhere else, but we really should `#include <vector>` here too.

Reply:
indeed. i thought it was in the header but it's not. added there.